### ISSUE TITLE
Lumen Install Issue

### ISSUE THREAD

**Issue by ReJust** [omniphx/forrest#148]
I know this has been mentioned earlier but I am having issues when installing on Lumen.  I followed the instructions exactly, however, it keeps throwing: Class redirect does not exist.  It is in reference to the BaseServiceProvider:

$input = new LaravelInput(app('request'));

Thoughts?

![screen shot 2017-04-11 at 11 52 56 am](https://cloud.githubusercontent.com/assets/13007455/24920598/74b743c2-1ead-11e7-89f1-106ede512f65.png)

**Comment by tabirkeland** [comment]
I just ran into this issue as well.  Going to try to downgrade a version or two and see if it resolves it.

**Comment by ReJust** [comment]
Yes, I downgraded to version 2.3.0 and it worked fine for Lumen, prior to it requiring the redirect class in the base controller.

**Comment by tabirkeland** [comment]
It has to do with this line in the `BaseServiceProvider.php(70)`:

```php
$redirect = new LaravelRedirect(app('redirect'));
```

When I wonder if this would work:

```php
$redirect = new LaravelRedirect(redirect());
```

**Comment by fauzyhanif** [comment]
vbvbbb

**Comment by omniphx** [comment]
Hey guys, I'm working on a new release 2.4 #151 that will fix this issue. Sorry for the delay, been kinda swamped lately.
